src/bitset_util.c

```diff
@@ -553,6 +553,88 @@ static uint16_t vecDecodeTable_uint16[256][8] = {
 #endif
 
 #ifdef CROARING_IS_X64
+CROARING_TARGET_AVX512
+size_t bitset_extract_setbits_avx512(const uint64_t *words, size_t length, uint32_t *out,
```

Comment:
If, as I recommend, you check for VBMI2 then there are much faster and simpler techniques: 
https://lemire.me/blog/2022/05/10/faster-bitset-decoding-using-intel-avx-512/

Reply:
I agree, how about next release?